src/Microsoft.Android.Sdk.ILLink/Microsoft.Android.Sdk.ILLink.csproj

```diff
@@ -1,17 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Since this file isn't in a `.sln` anymore, should we make a new `.sln` file next to this project?

I think you could probably open a `Microsoft.Android.Sdk.ILLink.sln` if you have a system .NET 6 installed.

Reply:
I think it is not needed. I usually just open the .csproj in  VS when I need UI.